infrastructure/cdn-in-a-box/traffic_ops_data/profiles/020-ATS_MID_TIER_CACHE.json

```diff
@@ -15,7 +15,7 @@
 			"configFile": "records.config",
 			"name": "CONFIG proxy.config.config_dir",
 			"secure": false,
-			"value": "STRING /etc/trafficserver"
+			"value": "STRING /opt/trafficserver/etc/trafficserver"
```

Comment:
Does `t3c` not support installation to `/`?

Reply:
It does, but CDN in a Box also should use a newer ATS version (currently using ATS 7.1.4), and building `./pkg -vo ats` yields an ATS RPM that uses `/opt/trafficserver` as the TS root directory, so using `/opt/trafficserver` seems inevitable.

Reply:
that's unfortunate